roles/ansible-keylime-tpm20/tasks/ibm-tpm.yml

```diff
@@ -33,40 +33,47 @@
   environment:
     TPM2TOOLS_TCTI: "tabrmd:bus_name=com.intel.tss2.Tabrmd"
 
+- name: Ensure tpm_server is not running
+  command: pkill tpm_server
+  ignore_errors: yes
+
 - name: Start TPM Emulator
   command: /usr/local/bin/tpm_serverd
   become: yes
 
-- name: enable service tpm2_abrmd and ensure it is not masked
+- name: Enable tpm2_abrmd service and ensure it is not masked
   systemd:
     name: tpm2-abrmd
     enabled: yes
     masked: no
 
-- name: Change TCTI to emulator
-  lineinfile:
-    path: /usr/lib/systemd/system/tpm2-abrmd.service
-    regexp: '^ExecStart='
-    line: ExecStart=/usr/sbin/tpm2-abrmd --tcti=mssim
+- name: Create a systemd drop-in directory for tpm2-abrmd
+  file:
+    dest: /etc/systemd/system/tpm2-abrmd.service.d/
+    state: directory
+    mode: '0777'
```

Comment:
Should this be `0755`?

Reply:
Yes probably, but I was following what already existed in `/etc/systemd/system`. All of the other directories there are `0777` but they are all `.wants` directories and there aren't any other drop-in dirs on the system. But yeah, starting more restrictive and then expanding it if we ever have a use case makes sense.